settings.gradle.kts

```diff
@@ -14,5 +14,6 @@ includeBuild("kalium") {
     // This dependency substitution should not be done on release mode once the Kalium library has been published to Maven repo
     dependencySubstitution {
         substitute(module("com.wire.kalium:kalium-network")).using(project(":network"))
```

Comment:
Shouldn't we need to remove the network module and leave only the `kalium-logic` as the only entry point from external consumers of Kalium?

Reply:
I guess we can. I just didn't want to check if `kalium-network` was being used when I passed through this file 😛 

I'll remove it